core/src/main/java/org/infinispan/interceptors/impl/DistCacheWriterInterceptor.java

```diff
@@ -88,7 +91,9 @@ public Object visitPutMapCommand(InvocationContext ctx, PutMapCommand command) t
          PutMapCommand putMapCommand = (PutMapCommand) rCommand;
          LocalizedCacheTopology cacheTopology = dm.getCacheTopology();
          Map<Object, Object> map = putMapCommand.getMap();
-         int count = 0;
+
```

Comment:
please refactor this whole lambda into `handlePutMapCommandReturn` and store `this::handlePutMapCommandReturn` in the field of the same name...